pkg/console/starter/starter.go

```diff
@@ -108,13 +111,37 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		oauthinformers.WithTweakListOptions(tweakListOptionsForOAuth),
 	)
 
-	recorder := ctx.EventRecorder
+	resourceSyncerInformers := v1helpers.NewKubeInformersForNamespaces(
+		kubeClient,
+		api.TargetNamespace,
+		api.OpenshiftConfigManagedNamespace,
+		api.OpenShiftConsoleOperatorNamespace,
+	)
 
 	operatorClient := &operatorclient.OperatorClient{
 		Informers: operatorConfigInformers,
 		Client:    operatorConfigClient.OperatorV1(),
 	}
 
+	resourceSyncer := resourcesynccontroller.NewResourceSyncController(
+		operatorClient,
+		resourceSyncerInformers,
+		v1helpers.CachedSecretGetter(kubeClient.CoreV1(), resourceSyncerInformers),
```

Comment:
Did we need a secret as well?

Reply:
Ah, looks like this is not optional, the `secretsGetter` is a required parameter.  Even if we aren't syncing secrets.